README.md

```diff
@@ -44,12 +44,9 @@ After checking out the project, cd-ing into it, and setting up a Python3.7 virtu
 - `wget https://raw.githubusercontent.com/hubmapconsortium/prov-vis/master/src/schema.json -O node_modules/@hubmap/prov-vis/es/schema.json`
 - (Note: Error with @hubmap-prov-vis dependency. Manually add
 the schema.json until [this issue is resolved](https://github.com/hubmapconsortium/portal-ui/issues/139).)
-- `npm run dev-build`
 - `./quick-start.sh`,
 - update `app.conf` with the Globus client ID and client secret, and the Elasticsearch endpoint,
-- `./quick-start.sh` again,
-and visit [localhost:5000](http://localhost:5000), or append `?react`
-to a `/dataset` url to see the React version.
+- `./dev-start.sh` to run the webpack dev and flask servers and visit [localhost:5001](http://localhost:5001).
```

Comment:
I think 5001 is the new react dev server... Won't folks still go to flask, and then the requests for JS from there are handled by the react dev server?

Reply:
Oh: got down to this:
```
    proxy: {
      '!(/static/public//**/**.*)': {
        target: 'http://localhost:5000/',
      },
    },
```
The react will proxy to the flask? Adding a sentence here to explain that would be good.